test/absinthe/schema/sdl_render_test.exs

```diff
@@ -187,28 +182,47 @@ defmodule SdlRenderTest do
         arg :times, :integer, default_value: 10, description: "The number of times"
       end
     end
+
+    object :order do
+      field :id, :id
+      field :name, :string
+    end
+
+    object :category do
+      field :name, :string
+    end
+
+    union :search_result do
+      types [:order, :category]
+    end
```

Comment:
Yes, I ran into this a few times when building this out.. I tried hard to avoid needing the entire list of types when rendering because that makes it difficult to enable rendering just an individual type's SDL representation via `inspect`.

It seems like you've got that working out fine though since it's passing tests :)

Reply:
I'm seeing you handle rendering both `Blueprint.TypeReference.Name` and `Blueprint.TypeReference.Identifier` separately, nice

Reply:
Yeah what I did is basically fall back to doing what you had where we just camelize stuff and hope for the best if the type can't be found. This allows us to render simple fragments of a schema without needing the whole thing.